include/ignition/math/detail/Box.hh

```diff
@@ -113,6 +121,167 @@ T Box<T>::Volume() const
   return this->size.X() * this->size.Y() * this->size.Z();
 }
 
+//////////////////////////////////////////////////
+/// \brief Given a *convex* polygon described by the verices in a given plane,
+/// compute the list of triangles which form this polygon.
+/// \param[in] _plane The plane in which the vertices exist.
+/// \param[in] _vertices The vertices of the polygon.
+/// \return A vector of triangles and their centroids, or an empty vector
```

Comment:
it looks like the 2nd value in the `std::pair`, which has type `T` is either `+1` or `-1`,  depending on the orientation of the plane. I think that should be documented in the `\return` doc-string

Reply:
you were faster! b4f9ddce0441145a3401581c5f707c43bf71e09e